main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -664,7 +676,19 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
     }
 
     def Postfix: Rule1[ParsedAst.Expression] = rule {
-      Apply ~ zeroOrMore(optWS ~ "." ~ Names.Definition ~ ArgumentList ~ SP ~> ParsedAst.Expression.Postfix)
+      ArraySlice ~ zeroOrMore(optWS ~ "." ~ Names.Definition ~ ArgumentList ~ SP ~> ParsedAst.Expression.Postfix)
+    }
```

Comment:
I am a little bit confused that Arrays are handled here in the "precedence hierarchy", but vectors are handled much earlier. I would have thought they would appear close together?